### ISSUE TITLE
Suggestion: Different variations of prefabs

### ISSUE THREAD

**Issue by slambo311** [slayerofdrosophila/tf2mapper#2]
Maybe make it so if you press a prefab button on a tile that already has the voxel, it changes to a different type (and when it's done going through all the variations, _then_ it removes the prefab). Here's a list of variations I came up with.

- Stairs to the next floor: 1x1 ramp, 2x1 ramp, 1x1 stairs, 2x1 stairs
- Medkit/Healthkit: Small medkit, medium medkit, large medkit
- Ammo crate: Small ammo crate, medium ammo crate, large ammo crate
- Shutter door: Regular door, team-respective door, resupply locker

**Comment by slayerofdrosophila** [comment]
Good idea. I have been considering this myself, but I am unsure if this would make removing the prefab too much of a pain. Maybe this could be available through some toggle button. If I have the time I will work toward implementing some way of getting these variations.

While you're here: what do you mean about the stairs? I don't understand what a "2x1" ramp would be. Thanks for checking this out.

**Comment by slambo311** [comment]
A 2x1 ramp would be a ramp that just takes up one extra tile and is longer.
![image](https://user-images.githubusercontent.com/61334045/156000362-c6488baf-2776-4d59-8a60-c00f638729ae.png)
Here's a little graph I made. The red one on the left is a 1x1 ramp, and the blue one on the right is a 2x1 ramp.
